### PR TITLE
add vishnur's hackbar

### PR DESCRIPTION
**Pull request by Vishnurr2k01**
I add a file in hackbar

### CODE REVIEW

File: _hackbars/vishnur.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+open source is the best source to show one's talent.
+
+
+---
+
+## Leave some greenery for the future to enjoy the Scenery..
+
+---
+
+![Melting_Earth]({{site.baseurl}}/assets/images/vishnur.jpg)
```
Comment (tasnimzotder) [inline _hackbars/vishnur.md:17]: hey @Vishnurr2k01 Welcome to Hacktoberfest-2020
Can you please add the missing image `vishnur.jpg`?

**Comment by Vishnurr2k01** [comment]
how to add bro?? Jam new to github..could u please guide me??
Thank you

**Comment by tasnimzotder** [comment]
Sure. You can follow the steps to add an image -

1. Take an image (climate-change themed) and rename it `vishnur.jpg`
1. On your local machine go to the repository directory (folder) `hacktoberfest-2020`
1. Copy the image `vishnur.jpg` to the directory `/assets/images/`
1. Make a commit
1. That's it 🎉

**Comment by Vishnurr2k01** [comment]
thank you
.I will update it as soon as possible

**Comment by Vishnurr2k01** [comment]
its showing upload disabled..

**Comment by tasnimzotder** [comment]
@Vishnurr2k01 It is uploaded
image

**Review by tasnimzotder — APPROVED** [review]
👍

**Comment by tasnimzotder** [comment]
@Vishnurr2k01, Thanks for your contributions :)

**Comment by Vishnurr2k01** [comment]
thank you : )

**Pull request merged**